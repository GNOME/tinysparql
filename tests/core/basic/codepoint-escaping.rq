SELECT "\u03B1\\udddd🐩" ?t {
  ?u ns\u003Ap ?t
}
