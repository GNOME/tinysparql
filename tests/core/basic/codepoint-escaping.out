"α\udddd🐩"	"d:x ns:p"
